src/vs/workbench/contrib/inlineChat/browser/inlineChatStrategies.ts

```diff
@@ -129,6 +135,15 @@ export class PreviewStrategy extends EditModeStrategy {
 		// nothing to do
 	}
 
+	getWidgetPosition(initialRender: boolean, _range: Range, _response: SessionResponse): Position | undefined {
+		const viewModel = this._editor._getViewModel();
```

Comment:
You could also capture the cursor position during the constructor and always return that here.